src/Auth/AuthProvider.php

```diff
@@ -242,6 +243,22 @@ private function clientCredentials(): ?Credentials
         return $this->credentials;
     }
 
+    /**
+     * @return array<string, string>
+     */
+    private function getHeaders(): array
+    {
+        $headers = [
+            'Accept' => 'application/json',
+            'Content-Type' => 'application/json',
+            'Authorization' => 'Basic ' . base64_encode(sprintf('%s:%s', $this->clientId, $this->clientSecret))
+        ];
+        if ($this->userAgent !== null) {
```

Comment:
Koukni kdyžtak sem https://github.com/fakturoid/fakturoid-ruby/blob/main/lib/fakturoid/config.rb#L92 kde vyžadujeme email a tady pak vytváříme user agenta pokud se nepošle https://github.com/fakturoid/fakturoid-ruby/blob/main/lib/fakturoid/config.rb#L27-L33 nedávalo by smysl tam udělat něco takového?

Reply:
Tohle mi příjde spíš jako nějaká business logika trochu. Ale i tak, bych to raději viděl v nějakém Symfony Bundle nebo nette extension než v libce pro API. Já bych nejraději toho User agenta odsud odebral úplně a nechal to na výchozím nastavení http klienta. Co myslíš?

Reply:
Zase je to povinná náležitost API a v konfiguraci to tak působí líp, ale když to bude dobře zdokumentované, tak klidně 👍

Reply:
Btw občas nám píšou PHP lidi s dotazem, že ClientInterface neexistuje:

```php
$fManager = new \Fakturoid\FakturoidManager(
    new ClientInterface(), // PSR-18 client <-------- tutok
```

Nepochopí tedy, že si tam mají doplnit vlastní knihovnu a tu iniciovat, tak jim tedy pošleme link na packagist a tuhle omáčku kolem, pokud bys s rámci dokumentování useragenta ještě dokázal trochu ujasnit toto, tak by to bylo super (třeba example s Guzzle nebo nějakými nejpopulárnějšími clienty?).

Reply:
@ollie  super info, že vám s tím píšou. To chce zdokumentovat :) 

Pak teda bych rozšířil dokumentaci o inicializaci Symfony a Guzzle klientů vč. nastavení výchozí hlavičky a toho user agenta bych odebral z knihovny. 
oki?

Reply:
Za mě ok. :)